src/firebase/db.js

```diff
@@ -0,0 +1,25 @@
+import {db} from "../firebase/firebase";
+
+export const addUser = async (guest) => {
+  try {
+    db.collection("guests").doc(`${guest.firstName} ${guest.lastName}`).set(guest);
+    console.log("guest added!");
+  } catch (error) {
+    console.log(error);
```

Comment:
Something to plan for is what to do in the event of an error. It's not a matter of if but when it happens. You should have some strategy to deal with it